### DISCUSSION TITLE
Github student pack

### DISCUSSION THREAD

**Discussion by mogumogudj** [community/community#50743] [GitHub Education]
Hello, 

every time I try to sign up for the student pack on GitHub it keeps telling me this. 

![Screenshot 2023-03-22 at 20 02 14](https://user-images.githubusercontent.com/116001748/227014375-cb4a0a45-3170-4d16-876f-2663fff300c7.png)

I tried fixing it for 2 hours now.. 
Can anyone help.. it's important and I need it fast

**Comment by samus-aran** [comment]
Hi @mogumogudj 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Hope this helps! Let us know how you get along 👍

**Comment by mogumogudj** [comment]
Hello,

I've tried to change everything of my profile and it didn't work. Is there somehow you guys can let me in..?

Greetings,
Wolf
________________________________
